src/Raven.Server/Rachis/FollowerAmbassador.cs

```diff
@@ -492,14 +492,19 @@ private void NotifyOnException(ref bool hadConnectionFailure, string message, Ex
             var isGracefulError = IsGracefulError(e);
             if (hadConnectionFailure || isGracefulError == false)
             {
-                Status = AmbassadorStatus.FailedToConnect;
-                StatusMessage = $"{message}.{Environment.NewLine}" + e;
-                if (_engine.Log.IsInfoEnabled)
+                //We don't want to notify about cluster change every time we check the connection, only if the change is new
+                if (Status != AmbassadorStatus.FailedToConnect)
```

Comment:
Not sure about this change. This func is called every few seconds because we keep retrying to reconnect and fail, and it triggered the notify function chain needlessly. On the other hand, maybe these constant notifications to the studio are necessary in some scenarios.

Reply:
I manually checked the regular case of bringing down a node and putting it back up, both when in and out of the studio and it worked